src/app.py

```diff
@@ -53,12 +53,12 @@ def license_page():
     "entities": [
         {
             "entity_name_long": "first name",
-            "entity_name_short": "First Name",
+            "entity_name_short": "First Name - You can use % for a wildcard match",
```

Comment:
```suggestion
            "entity_name_short": "First Name - Start or end name with % for partial matches",
```
Thoughts on wording (to make it a bit more accessible). Same for below